.vscode/settings.json

```diff
@@ -0,0 +1,2 @@
+{
+}
```

Comment:
This file is still here; not sure if you meant to remove it

(It won't let me edit your PR directly so can't help sadly)